src/docs/user/references/DataConnectors.adoc

```diff
@@ -7,127 +7,212 @@ We begin by presenting FTP and SFTP connectors then, URL connector.
 | `<PROTOCOL>_EXTRACT_ARCHIVE`
 | Used only when `<PROTOCOL>_MODE=GET`. If set to `true`, the imported file will be extracted if it is an archive.
+| Workflow, Task
+| Yes
 |  Boolean [true or false]
+| false
 | `<PROTOCOL>_USERNAME`
 | Username to use for the authentication.
+| Workflow, Task
+| Yes
 |  String
+| e.g. \ftp://someuser@example.com
 |===
 
 *How to use this task:* The task requires the following third-party credential: `{key: <PROTOCOL>://<username>@<hostname>, value: <PROTOCOL>_PASSWORD}`. Please refer to the User documentation to learn how to add link:../user/ProActiveUserGuide.html#_third_party_credentials[third-party credentials].
 
 ==== URL connector
 
 The URL connector allows, in addition, to import data using HTTP and HTTPS protocols.
-*Task Variables:*
 
-.URL_Connector_Task variables
-[cols="2,5,2"]
+*Variables:*
+
+.URL Connector variables
+[cols="2,5,2,2,2,2"]
 |===
-| *Variable name* | *Description* | *Type*
+| *Variable name* | *Description* | *Level* | *Required?*  | *Type*  | *Default/Examples*
 | `FILE_URL`
 | Link to a file accessible using HTTP, HTTPS, SFTP or FTP protocols.
 
   FTP and SFTP urls must have the following patterns:
 
-- `\ftp://<username>:<password>@<hostname>:<port>/<path>`
+- `\ftp://<username>[:<password>]@<hostname>[:<port>]/<relativePath>`
 
-- `sftp://<username>@<host>[:<port>][/<directory>]`
+- `sftp://<username>[:<password>]@<hostname>[:<port>]/<relativePath>`
 
+| Task
+| Yes
 | String
+| e.g. sftp://user:pass@example.com/test.txt
 | `FTP_LOCAL_RELATIVE_PATH`
 |  Local relative path from which we upload (or to which we download) file(s).
-  LOCAL_RELATIVE_PATH can contain either a path to a file, a directory terminated by / or an empty value for the root.
+  LOCAL_RELATIVE_PATH can contain either a path to a file, a directory terminated by `/` or an empty value for the root.
+| Task
+| No
 | String
+| e.g. localDirectory/, example.zip
 | `EXTRACT_ARCHIVE`
 | If set to `true`, the imported file will be extracted if it is an archive.
+| Task
+| Yes
 | Boolean [true or false]
+| false
 |===
 
+*How to use this task:* We highly recommend the user to not provide his password within the URL and to instead use the third-party credential mechanism as follows: `{key: <PROTOCOL>://<username>@<hostname>, value: <PROTOCOL>_PASSWORD}`. Please refer to the User documentation to learn how to add link:../user/ProActiveUserGuide.html#_third_party_credentials[third-party credentials].
+
 === SQL
 The SQL connectors allow to import data from Relational DataBase Management Systems (RDBMS).
 Currently, we have connectors for Mysql, Oracle, Postgres and Sql server.
 
-*Task Variables:*
+*Variables:*
 All SQL connectors share the same list of variables. Consequently, we describe them in the following table using a unique notation.
 `<RDBMS_NAME>` can take one of the following values: {`POSTGRES`, `ORACLE`, `SQL_SERVER`, `MYSQL`}
 
-.SQL_Connector_Task variables
-[cols="2,5,2"]
+.SQL Connector_Task variables
```

Comment:
Is it intended to keep task here when removing others ?

Reply:
Not at all. It is a mistake. Thank u for the observation.